src/main/java/com/vinylteam/vinyl/dao/jdbc/JdbcUserDao.java

```diff
@@ -1,6 +1,89 @@
 package com.vinylteam.vinyl.dao.jdbc;
 
 import com.vinylteam.vinyl.dao.UserDao;
+import com.vinylteam.vinyl.dao.jdbc.mapper.UserRowMapper;
+import com.vinylteam.vinyl.entity.User;
+import org.postgresql.ds.PGSimpleDataSource;
+import org.postgresql.util.PSQLException;
+
+import java.sql.*;
 
 public class JdbcUserDao implements UserDao {
+
+    private final PGSimpleDataSource dataSource;
+    private final String COUNT_ALL = "SELECT COUNT(*) FROM public.users";
+    private final String FIND_BY_EMAIL = "SELECT * FROM public.users" +
+            " WHERE email=?";
+    private final String INSERT = "INSERT INTO public.users" +
+            " (\"email\", \"password\", \"salt\", \"iterations\", \"role\")" +
+            " VALUES (?, ?, ?, ?, ?)";
+
+    public JdbcUserDao(PGSimpleDataSource dataSource) {
+        this.dataSource = dataSource;
+    }
+
+    public boolean add(User user) {
+
+        boolean isAdded = false;
```

Comment:
Is initialization is required here?

Reply:
It's not. I just thought it'd be clearer this way.
Should I remove it?

Reply:
Yes